fixtures/gists-app/app/routes/meta.tsx

```diff
@@ -0,0 +1,28 @@
+import { LoaderFunction, MetaFunction } from "remix";
```

Comment:
I think we’re trying to use Ryan’s `createFixture` utility as much as possible instead of the gist app. Could you switch this test to use that instead?